examples/docker/00-hello-world/go.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/bin/bash -e
```

Reply:
I'll put in my standard bash strict boilerplate

Reply:
Done